Accepted-RFCs/RFC-011-news-announcements.md

```diff
@@ -0,0 +1,165 @@
+
+Unlike how we handle the blog, there wouldn't be any commenting feature. 
+
+Unlike how we handle the blog, there wouldn't be any banner or hero images.
+
+Because news is timely, the individual post URLs would use the date rather than a title.
+
+### Why not use the blog?
+
+Two reasons:  First, it would split the purpose and publishing resources of the blog. The blog's purpose is supporting docs advocacy and education.
+
+Second, while the framework might be similar, it isn't exactly the same. So we can reuse a lot of the work on the blog, but it has different content needs.
+
+### "Creating" the News Team
+
+The team that would handle this would be the "News team," and essentially be our PR team. But functionally, that doesn't end up being a new team, as we already have a de facto PR team between those who run our social media (namely Felicity and Carrie), people planning Content Strategy, and the co-chairs.
+
+This team would just be a formalization of who vets news posts before going on, to make sure news is on message. The ideal people for this are journalism and communication studies nerds. We happen to already be well stocked in that regard. :)
+
+### Forward facing: emailing newsletters
+
+If we want to get to a point of having a mailing list to keep our users informed, this is the content we'd use. And by having two categories, people could opt into different types of content. However, that isn't a requirement to execute on this idea.
+
+## Consequences
+
+This allows us a space to write about upcoming changes for users to experiment with (such as if we start playing in the space of having "dev branches"), announcements for recruiting, celebrating our contributors' efforts elsewhere (like announcing them being Write the Docs speakers), and upcoming meetups (like holding writing days at Write the Docs).
+
+This would be an additional effort set for the Tech Team, to take the blog and tweak it for another subfolder. That might involve refactoring material to be shared, or not—details on the code execution part of this effort is beyond the scope of the RFC, barring noting any concerns, issues, or plans that the Tech Team might share upon reviewing this.
+
+This could also play into future endeavors as we potentially expand, such as announcing Good Docs-hosted talks (which are themselves ways of passive recruiting).
+
+And this would be some additional ongoing effort on the part of the News team, though the effort should be light, and the payoff for that effort well worth it.
+
+## Links and prior art
+
+[How Write the Docs does their newsletters](https://www.writethedocs.org/blog/archive/tag/newsletter/), which as an index page feels like it works as email-delivered newsletters first, site-formatted news second.
+
+## Open questions
+
+{This section is optional and is where you can list questions that won't be resolved by this RFC, including those raised in comments from community members.}
+
+
+## Decisions deferred
```

Comment:
I feel like the first three bullet points below are in fact the "next steps" rather than "deferred decisions." It's an arbitrary distinction, but these are things that need to be addressed in order to take the implementation described in an approved RFC into production.